homeassistant/components/arcam_fmj/__init__.py

```diff
@@ -107,13 +125,15 @@ async def async_setup_entry(hass: HomeAssistantType, entry: config_entries.Confi
             {CONF_HOST: entry.data[CONF_HOST], CONF_PORT: entry.data[CONF_PORT]}
         ),
     )
+    tasks = hass.data.setdefault(DOMAIN_DATA_TASKS, {})
 
     hass.data[DOMAIN_DATA_ENTRIES][entry.entry_id] = {
         "client": client,
         "config": config,
     }
 
-    asyncio.ensure_future(_run_client(hass, client, config[CONF_SCAN_INTERVAL]))
+    task = asyncio.ensure_future(_run_client(hass, client, DEFAULT_SCAN_INTERVAL))
```

Comment:
Please use `asyncio.create_task` instead of `asyncio.ensure_future`.

Reply:
Probably never. Just defensive coding. But I agree, no point in protecting.